R/background.R

```diff
@@ -107,15 +109,17 @@ rcmdcheck_process <- R6Class(
 #' @importFrom callr rcmd_process rcmd_process_options
 #' @importFrom desc desc
 
-rcc_init <- function(self, private, super, path, args, libpath, repos) {
+rcc_init <- function(self, private, super, path, check_args = character(),
+                     build_args = character(), libpath, repos) {
```

Comment:
Is this called from somewhere with `check_args` and `build_args` set to anything but an empty vector?

Reply:
This is also solved by moving the named args of `check_args()` and `build_args()` to the `initialize` function def -> then we can simply use unnamed args here.